packages/shared/src/components/onboarding/FilterOnboarding.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { cloudinary } from '../../lib/image';
+import OnboardingStep from './OnboardingStep';
+import FeedTopicCard from '../containers/FeedTopicCard';
+import { TagCategory } from '../../graphql/feedSettings';
+import { OnboardingFiltersLayout } from '../../lib/featureValues';
+
+interface FilterOnboardingProps {
+  tagsCategories: TagCategory[];
+  selectedId: Record<string, boolean>;
+  topicLayout?: OnboardingFiltersLayout;
+  onSelectedChange: (id: string) => void;
+}
+
+const classes: Record<OnboardingFiltersLayout, string> = {
+  grid: 'grid-cols-3 gap-6',
+  list: 'grid-cols-1 gap-4',
+};
+
+function FilterOnboarding({
+  selectedId,
+  topicLayout,
+  tagsCategories,
+  onSelectedChange,
+}: FilterOnboardingProps): ReactElement {
+  return (
+    <OnboardingStep
+      topIcon={
+        <img
+          className="mx-auto mb-6 w-16"
+          src={cloudinary.feedFilters.supercharge}
+          alt="test"
```

Comment:
Please change this before merging, else we'll forget about it.

Reply:
Oh, no. Thank you for pointing that out. Didn't realize this mistake. Let me refactor it now.